match/lib/match/storage/google_cloud_storage.rb

```diff
@@ -181,7 +181,7 @@ def skip_docs
       end
 
       def generate_matchfile_content
-        return "bucket_name(\"#{self.bucket_name}\")"
+        return "google_cloud_bucket_name(\"#{self.bucket_name}\")"
```

Comment:
Ah shoot! Was this really wrong? 😱

Reply:
It would be worth double-checking but I had to change my `Matchfile` to this before it would work.

Reply:
That this didn't pop up earlier also confused me, but checking the documentation it is indeed `google_cloud_bucket_name`.